lib/apiservers/portlayer/restapi/handlers/storage_handlers.go

```diff
@@ -648,3 +788,25 @@ func createVsphereVolumeStore(op trace.Operation, dsurl *url.URL, name string, h
 	}
 	return vs, nil
 }
+
+func decodeFilterSpec(filterSpec *string) *archive.FilterSpec {
```

Comment:
Hmmm, the personality is producing an encoded string of the json unmarshal. Check the personality side.